libs/ml/tests/ml/layers/PRelu.cpp

```diff
@@ -230,14 +230,16 @@ TYPED_TEST(PReluTest, saveparams_test)
   TypeParam loss = layer.Evaluate(error_output);
   layer.BackPropagate(error_output);
   layer.ApplyRegularisation();
-  layer.Step(DataType{0.1f});
```

Comment:
With Step there was the option to set the learning rate - is there an equivalent with GetGradients + ApplyGradients?

Reply:
no. but there shouldn't be.
This is because Step was effectively doing SGD without pointing that out explicitly.

If we really want to do SGD without using the SGDOptimiser there's nothing that stops us doing what @MissingNO57 proposed (e.g. graph.ApplyGradients(-LEARNING_RATE * graph.GetGradients())), but then at least we're being explicit about what we're doing.